.github/workflows/main.yaml

```diff
@@ -101,6 +101,8 @@ jobs:
       group: publish
     steps:
       - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
Just out of curiosity, why do we have to add this?

Reply:
The default value is set to `1` (See https://github.com/actions/checkout) and that is fine for most jobs except for the one that builds the docs because we use an mkdocs plugin that sets the creation and last update times in the lower left corner of the documentation pages based on git history. If it cannot determine the correct dates then it just uses the current date as shown in the image below.

![image](https://github.com/aai-institute/pyDVL/assets/27914730/441f6a6e-067a-4d45-a666-c60968571be4)

Reply:
Thanks for the explanation:)